### ISSUE TITLE
Academic dates block uses incorrect card fields creating an accessibility violation

### ISSUE THREAD

**Issue by bspeare** [uiowa/uiowa#8495]
# Problem
Academic dates block uses the subtitle field for card content creating an accessibility violation when it should be using the content field.

# Steps to reproduce
https://grad.uiowa.edu/academics/deadlines 

![Image](https://github.com/user-attachments/assets/1bc3beac-73b0-473a-a37c-f08538d6efcb)

# Proposed solution
Update the block to use the correct card markup.
